server/routes/room_routes.py

```diff
@@ -0,0 +1,18 @@
+from flask import Blueprint
+
+from controller.room_controller import RoomController
+
+room_bp = Blueprint('room_bp', __name__)
+
+@room_bp.route('/room',methods=['GET'])
+def get_room_details():
+    return RoomController().get_room_details()
+    
+@room_bp.route("/get-all-rooms", methods = ['GET'])
```

Comment:
It is considered good practice to avoid using hyphens in URL endpoints if possible.  we need ensure that the URL is clear, concise, and easy to understand for users.

So we can change this route to `/rooms`